example/js/lib/api/FixedDecimalGroupingStrategy.d.ts

```diff
@@ -4,13 +4,12 @@ import type { pointer, codepoint } from "./diplomat-runtime.d.ts";
 
 
 export class FixedDecimalGroupingStrategy {
-    
 
-    static fromValue(value : FixedDecimalGroupingStrategy | string) : FixedDecimalGroupingStrategy; 
+    static fromValue(value : FixedDecimalGroupingStrategy | string) : FixedDecimalGroupingStrategy;
```

Comment:
issue: you removed the space in front of the colon for some methods but not others